Doc/library/enum.rst

```diff
@@ -207,8 +207,8 @@ Data Types
 
    .. method:: EnumType.__dir__(cls)
 
-      Returns ``['__class__', '__doc__', '__members__', '__module__']`` and the
-      names of the members in *cls*::
+      Returns ``['__class__', '__contains__', '__doc__', '__getitem__', '__iter__', '__len__', '__members__', '__module__', '__name__', '__qualname__']``
+      and the names of the members in *cls*::
```

Comment:
I wonder if there's a better way to talk about this now that the list is longer. I'd certainly mention the names first and the special methods second.